### DISCUSSION TITLE
Help me with that error ugently.

### DISCUSSION THREAD

**Discussion by do-work97** [DarkaOnLine/L5-Swagger#576] [General]
- L5-Swagger Version: #.#.# (`composer show | grep l5-swagger`)
- PHP Version (`php -v`): 8.2.12
- OS: windows
-laravel/framework 9.52.16
### Description:
I will try to fix that error accoding your tips, but i can't solve that problem.

### Steps To Reproduce:

![Screenshot 2024-01-14 152703](https://github.com/DarkaOnLine/L5-Swagger/assets/153564932/f5993c5b-94fd-478b-ab49-4ab0c1c3509c)

**Comment by prialgauskaslt** [comment]
here you go: https://letmegooglethat.com/?q=proxy+laravel+csrf+token+mismatch+VerifyCsrfToken.php
